analyzer/runtime/runtime.go

```diff
@@ -289,18 +289,34 @@ func (m *Main) queueDbUpdates(batch *storage.QueryBatch, data *BlockData) {
 		for addr := range transactionData.RelatedAccountAddresses {
 			batch.Queue(queries.RuntimeRelatedTransactionInsert, m.runtime, addr, data.Header.Round, transactionData.Index)
 		}
+		var error_module string
+		var error_code uint32
+		var error_message *string
+		if transactionData.Error != nil {
+			error_module = transactionData.Error.Module
+			error_code = transactionData.Error.Code
+			error_message = transactionData.Error.Message
+		}
 		batch.Queue(
 			queries.RuntimeTransactionInsert,
 			m.runtime,
 			data.Header.Round,
 			transactionData.Index,
 			transactionData.Hash,
 			transactionData.EthHash,
+			transactionData.Fee.String(),
+			transactionData.GasLimit,
 			transactionData.GasUsed,
 			transactionData.Size,
 			data.Header.Timestamp,
-			transactionData.Raw,
-			transactionData.RawResult,
+			transactionData.Method,
+			transactionData.Body,
+			transactionData.To,
+			transactionData.Amount,
```

Comment:
This conversion finally works "automatically" (we do provide a special serialization method on BigInt) as it should since we've migrated to pgx v5. (Edit: Not really, read on)

If you're alluding to `transactionData.Fee.String()` above, that did require the `String()` for some reason ... The difference is that `Fee` is `BigInt` whereas `Amount` is `*BigInt`. I change `Fee.String()` to `&Fee` and now everything (still) works, with no explicit `String()`. :grimacing: